docs/FAQ/how-to-deploy-bold-bi-and-bold-reports-in-an-ack-cluster.md

```diff
@@ -0,0 +1,337 @@
+
+# Deploy Bold BI and Bold Reports using Kubectl
+
+The following steps will guide you through the process of deploying Bold BI and Bold Reports using kubectl in an Alibaba Cloud Kubernetes (ACK) cluster.
+
+## Deployment prerequisites
+
+The following requirements are necessary to deploy the Bold BI and Bold Reports solution using kubectl:
+
+* [Install Kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl): Make sure you have kubectl installed on your local machine to facilitate the deployment process.
+* [NAS File System](../pre-requisites.md#ack-file-sotrage): Set up a NAS file system to store the required data for Bold BI and Bold Reports.
+* [Create and connect a ACK cluster](../pre-requisites.md#ack-cluster): Create and connect an ACK cluster in Alibaba Cloud that will host the Bold BI and Bold Reports deployment.
+* Load Balancing- [Nginx](https://kubernetes.github.io/ingress-nginx/deploy/) : By default, an Alibaba Cloud Kubernetes (ACK) cluster has Nginx already installed in the kube-system namespace, so there is no need to manually install it. The Nginx Ingress Controller is available and can be utilized to enable load balancing for your Bold BI and Bold Reports deployment.
+
+Please ensure that you have fulfilled these prerequisites before proceeding with the deployment.
+
+## Steps for Bold BI and Bold Reports auto deployment using kubectl.
+
+1. Download the deployment file from [here](https://raw.githubusercontent.com/boldbi/boldbi-kubernetes/239310-Bold-BI-Alibaba-Cloud-Kubernetes-(ACK)-Doc/deploy/deploy/common_ack_deploy.yaml) to deploy Bold BI and Bold Reports on AKS.
+
+2. Navigate to the folder where the deployment files were downloaded in **Step 1**.
+
+3. Follow the documentation provided at [this link](https://www.alibabacloud.com/help/en/nas/latest/create-a-nas-file-system) to create a `file system` in Alibaba Cloud.
+
+4. Once the file system is created, locate and click on the cube icon. This will allow you to view the mount target hostname. Take note of the mount target hostname, as you will need it for mounting the file system with our application.
+
+   ![NAS-Mount-Target](images/Nas-mount-target.png)
+
+5. Open the **common_ack_deploy.yaml** file and replace the <host_name_of_mount_target> placeholder with the actual mount target hostname noted in the previous step.
+
+    ![PV Claim](images/ack_pv_claim.png)
+
+6. If you have a DNS to map with the application, proceed to the next steps, otherwise, skip to **Step 10**. 
+
+7. In common_ack_deploy.yaml file and locate line 2335. Uncomment the host value and replace example.com with your DNS hostname.
+
+      ![DNS](images/ack_ingress_http_yaml.png)
+  
+8. If you have an SSL certificate for your DNS and need to configure the site with it, follow these steps; otherwise, skip to **Step 10**.
+
+9. In the ingress spec of the common_ack_deploy.yaml file, uncomment the tls section. Replace example.com with your DNS hostname and save the file.
```

Comment:
9. In the ingress spec of the common_ack_deploy.yaml file, uncomment the tls section. Replace example.com with your DNS hostname and save the file.